### DISCUSSION TITLE
Verify Student Account

### DISCUSSION THREAD

**Discussion by snannra** [community/community#109641] [GitHub Education]
### Select Topic Area

General

### Body

I'm a student in University and I'm looking to gain access to copilot. I've seen that as long as I'm a student, I should be able to get to use copilot for free. I was wondering how do I verify my account to show that I'm a student, so that I can gain access to copilot for free.

**Comment by namith1003** [comment]
you can apply for the student pack

https://education.github.com/discount_requests/application

you get copilot as part of the pack for free but it may take a few days to fully approve and will require proof of documents to apply for, which I'm sure you will have so don't worry.

Hope this answers your question.

**Comment by dyardy** [comment]
I have been approved for student access but yet to get the copilot access on my account.  Do I need to do something else?

**Comment by byteakp** [comment]
ME too
